panel/param.py

```diff
@@ -260,6 +293,91 @@ def _get_root(self, doc, comm=None):
 
 
 
+class ParamMethod(PaneBase):
+    """
+    ParamMethod panes wrap methods annotated with the param.depends
+    decorator and rerenders the plot when any of the methods parameters
```

Comment:
It's not required for them to be annotated, right?  (Given that the default behavior is to watch all parameters.)

Reply:
"methods" -> "method's"